config/default.json

```diff
@@ -14,7 +14,7 @@
     "ccdComponentApi": "https://ccd-api-gateway-web-prod.service.core-compute-prod.internal/health",
     "ccdDataApi": "http://ccd-data-store-api-prod.service.core-compute-prod.internal/health",
     "documentsApi": "http://dm-store-prod.service.core-compute-prod.internal/health",
-    "em_hrs_api": "http://em-hrs-api-prod.service.core-compute-prod.internal/health",
+    //"em_hrs_api": "http://em-hrs-api-prod.service.core-compute-prod.internal/health",
```

Comment:
json doesn’t support comments? unless you're pre-processing it?

Reply:
Good spot!